bin/zopen-query

```diff
@@ -61,7 +61,7 @@ colorizepct()
   elif [ ${percentage} -eq 100 ]; then
     colored="${GREEN}"
   elif [ ${percentage} -gt 50 ]; then
-    colored="${BLUE}"
+    colored="${BOLD}${BLUE}"
```

Comment:
Is the blue more recognizable when bold?

Reply:
To my eyes, yes - it's brighter